spec/requests/discourse_needs_love/needs_love_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+module DiscourseNeedsLove
+  describe NeedsLoveController do
+    let(:topic) { Fabricate(:topic) }
+    let(:post) { Fabricate(:post, topic: topic) }
```

Comment:
```suggestion
    fab!(:topic) { Fabricate(:topic) }
    fab!(:post) { Fabricate(:post, topic: topic) }
```

is it possible to use `fab!` here?